packages/web/src/components/data-entry/FormTile.module.css

```diff
@@ -184,6 +184,14 @@
   font-weight: var(--font-demi-bold);
 }
 
+.trackAvailabilityButton {
+  height: 35px !important;
```

Comment:
any chance we can lose the `!important` by using more specific selectors or something?